ipaserver/install/httpinstance.py

```diff
@@ -125,9 +125,22 @@ def __init__(self, fstore=None, cert_nickname='Server-Cert',
         self.cert_nickname = cert_nickname
         self.ca_is_configured = True
         self.keytab_user = constants.GSSPROXY_USER
+        self.subject_base = None
 
     subject_base = ipautil.dn_attribute_property('_subject_base')
 
+    def _get_certdb(self, create=False):
```

Comment:
It would be much easier to just wait for mod_nss -> mod_ssl migration patches to be pushed and then just remove all of the NSS-based cruft from here,

Reply:
With mod_ssl, we still need to use some of the new features for migration. NSS auto-migrated databases like HTTPD's alias DB.

Reply:
@rcritten told me that #1449 PR is almost ready. It doesn't make much sense to disrupt the other PR with my changes.